sentry/templates/configmap-symbolicator.yaml

```diff
@@ -14,8 +14,21 @@ data:
     cache_dir: "/data"
     bind: "0.0.0.0:{{ template "symbolicator.port" }}"
     logging:
-      level: "{{ .Values.symbolicator.api.logging.level }}"
+      level: {{ .Values.symbolicator.api.logging.level | quote }}
     metrics:
-      statsd: null
-    sentry_dsn: null # TODO: Automatically fill this with the internal project DSN
+    {{- if .Values.symbolicator.api.statsd.server }}
```

Comment:
Maybe we should make this logic super simple by just passing a yaml from the values file ?
With toYaml

It feels we need to add each config in the repository every time a new one is available.

Reply:
Actually true. Will update the PR in next days

Reply:
@Mokto I updated the PR. Symbolicator starts fine with the default values config and also works for me when passing custom config.
should be a backwards compatible change as the defaults didnt change